Add more delay between SPI transactions when not in block mode.
EKH01 in Openwrt has less delay between SPI transactions than Buildroot.
Currently the driver has enough delay between blocks but not when the transaction isn't a block.
Banff needs more delay than Eagle Crest, that's why this issue wasn't happening on Eagle Crest.

diff --git a/spi.c b/spi.c
index 5863943..05f88ad 100644
--- a/spi.c
+++ b/spi.c
@@ -563,7 +563,7 @@ static int morse_spi_cmd53_read(struct morse_spi *mspi, u8 fn, u32 address, u8 *
 
 	if (!block) {
 		/* Scale bytes delay to block */
-		u32 extra_bytes = (count * mspi->inter_block_delay_bytes) / MMC_SPI_BLOCKSIZE;
+		u32 extra_bytes = max(250, (count * mspi->inter_block_delay_bytes) / MMC_SPI_BLOCKSIZE);
 
 		/* Allow 4 bytes for CRC and another 10 bytes for start block token & chip delays
 		 * (usually comes in 2).
@@ -692,7 +692,8 @@ static int morse_spi_cmd53_write(struct morse_spi *mspi, u8 fn, u32 address, u8
 		cp += sizeof(crc);
 
 		/* Allow more bytes for status and chip processing (depends on CLK) */
-		cp += block ? mspi->inter_block_delay_bytes : 4;
+		cp += block ? mspi->inter_block_delay_bytes
+					: max(250,(count * mspi->inter_block_delay_bytes) / MMC_SPI_BLOCKSIZE);
 	}
 
 	if (enable_ext_xtal_init) {
